src/MISC/fix_eco_force.cpp

```diff
@@ -0,0 +1,642 @@
+/* ----------------------------------------------------------------------
+ LAMMPS - Large-scale Atomic/Molecular Massively Parallel Simulator
+ http://lammps.sandia.gov, Sandia National Laboratdir_veces
+ Steve Plimpton, sjplimp@sandia.gov
+
+ Copyright (2003) Sandia Corporation.  Under the terms of Contract
+ DE-AC04-94AL85000 with Sandia Corporation, the U.S. Government retains
+ certain rights in this software.  This software is distributed under
+ the GNU General Public License.
+
+ See the README file in the top-level LAMMPS directory.
+ ------------------------------------------------------------------------- */
+
+/* ----------------------------------------------------------------------
+   Contributing authors: Adrian A. Schratt and Volker Mohles (ICAMS)
+------------------------------------------------------------------------- */
+
+#include "math.h"
```

Comment:
please change the include statements to follow our conventions as described in `doc/include-file-conventions.md`